src/vanilla.ts

```diff
@@ -288,9 +286,10 @@ const buildProxyFunction = (
         initialObject,
         key
       ) as PropertyDescriptor
-      if (desc.get || desc.set) {
-        Object.defineProperty(baseObject, key, desc)
-      } else {
+      const hasValue = 'value' in desc
+      delete desc.value
```

Comment:
@dai-shi just curious, by why `delete desc.value` and then reset it?

I had noticed when first diving into this that `proxyObject[key] = initialObject[key]` invoked the handler's `set` and did the whole "version"/"notifyUpdate"s process for every key on `initialObject`.

My curiosity was whether, since we've _just_ created the proxy, there should defacto be no subscribers, maybe we could skip all of that?

I might be missing something else important that the `handler.set` does, not sure.

Reply:
AFAIR, some tests failed.

Reply:
Ah sure, it looks like it sets up child proxies (just glancing at the `subscribe.test.tsx` failure). That makes sense.

Maybe leave an in-source hint for future maintainers/newbies? LIke

```
// Unset and reset the value to ensure child proxies are wrapped
const hasValue = 'value' in desc
delete desc.value
```

Reply:
399f92d

Reply:
Great, thanks!